snap/snapcraft.yaml

```diff
@@ -526,43 +547,72 @@ parts:
   # from edgex-go which at this point will fail
   # eventually we would like these changes to be upstreamed but until then
   # they live here. see also https://github.com/edgexfoundry/edgex-go/issues/1109
-  go:
+  go112:
```

Comment:
why still need go112?

Reply:
Because the snap builds consul and vault from source, and vault 1.3.1 can only be built with go 1.12.